src/Task/Assets/Less.php

```diff
@@ -112,7 +112,7 @@ public function compiler($compiler, array $options = [])
     protected function compile($file)
     {
         if (is_callable($this->lessCompiler)) {
-            return $this->lessCompiler($file, $this->compilerOptions);
+            return call_user_func($this->lessCompiler, $file, $this->compilerOptions);
         }
         if (method_exists($this, $this->lessCompiler)) {
             return $this->{$this->lessCompiler}($file);
```

Comment:
$this->compilerOptions also should be passed here (in case with `lessphp`):
```php
return $this->{$this->lessCompiler}($file, $this->compilerOptions);
```